bin/propolis-standalone/src/main.rs

```diff
@@ -863,6 +863,32 @@ fn setup_instance(
     Ok((inst, com1_sock))
 }
 
+/// Check bhyve and viona API versions, squawking if they do not meet
+/// expectations, but ultimately still allowing forward progress since
+/// propolis-standalone lives in the Thunderdome.
+fn api_version_checks(log: &slog::Logger) -> std::io::Result<()> {
+    match api_version::check() {
+        Err(api_version::Error::Io(e)) => {
+            // IO errors _are_ fatal
+            return Err(e);
+        }
+        Err(api_version::Error::Mismatch(comp, act, exp)) => {
+            // Make noise about version mismatch, but soldier on and let the
+            // user decide if they want to quit
+            slog::error!(
+                log,
+                "{} API version mismatch {} != {}",
+                comp,
+                act,
+                exp
+            );
+        }
+        _ => {}
```

Comment:
No other variants right? I'd just remove to make sure any new additions don't silently pass here.

Reply:
Filled out w/ the Ok case so new error cases will result in attention being required